doc/man3/SSL_get_verify_result.pod

```diff
@@ -56,7 +61,7 @@ L<openssl-verify(1)>
 
 =head1 COPYRIGHT
 
-Copyright 2000-2016 The OpenSSL Project Authors. All Rights Reserved.
+Copyright 2000-2018 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
2018 : need an update

Reply:
Yup, this PR has been outstanding since 2018...